templates/experiments/experiment_form.html

```diff
@@ -129,9 +129,12 @@ <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
       {% flag "experiment_versions" %}
         <button type="submit" class="pg-button-secondary" name="action" {% if disable_version_button %}disabled="True"{% endif %} value="save_and_version">Create Version</button>
       {% endflag %}
-      <a href="{% url 'experiments:delete' team.slug experiment.id %}" class="pg-button-danger" onclick="return confirm('Are you sure you want to delete this experiment? This action cannot be undone.')">
-        Delete
-      </a>
+      <form action="{url 'experiments:archive' team_slug=team.slug pk=experiment.id}" method="post" onsubmit="return confirm('Are you sure you want to archive this experiment?');">
```

Comment:
keeping as a draft until this is functional-- url works but something is wrong with the from since adding a "dummy" form before this one make this one work